google-cloud-debugger/lib/google/cloud/debugger/breakpoint/variable.rb

```diff
@@ -102,6 +102,10 @@ class Variable
           BUFFER_FULL_MSG = "Buffer full. Use an expression to see more data."
 
           ##
+          # @private Error message when variable can't be converted.
+          FAIL_CONVERSION_MSG = "Error: Variable couldn't be evaluated."
```

Comment:
How about "Error: Value couldn't be displayed." Because in this case the evaluation succeeds but conversion to a displayable string or structure failed.